lib/confirmation_sender.rb

```diff
@@ -0,0 +1,5 @@
+module ConfirmationSender
```

Comment:
@DavidKnott I would suggest to refactor all two factor authentication related methods into a two factor authentication service. Similar to what we have done in class for calling external APIs.
What you do you think?
Adding @cews7 @ethanbennett